feat(custom-extensions): add custom extensions in the theme (#15)
erickzanardo/nes_ui: README.md

More to come!

## Custom Extension

> Nes UI has built taking advantage of the Theme Extension, so that means, you can also provide your own extension.

Reviewer: ```suggestion
Nes UI is built taking advantage of the Theme Extensions and custom extensions can also be provided.
```
